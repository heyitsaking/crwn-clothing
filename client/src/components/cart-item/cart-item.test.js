import React from "react";
import { shallow } from "enzyme";

import CartItem from "./cart-item.component";

it("hould render CartItem component", () => {
  const mockItem = {
    imageUrl: "www.testlink.com",
    price: 10,
    name: "jackets",
    quantity: 3,
  };

  expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});
