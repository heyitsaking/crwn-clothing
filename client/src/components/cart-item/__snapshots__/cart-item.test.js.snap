// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hould render CartItem component 1`] = `ShallowWrapper {}`;
